src/Symfony/Component/Workflow/Registry.php

```diff
@@ -82,6 +82,11 @@ public function all($subject): array
         return $matched;
     }
 
+    /**
+     * @param WorkflowSupportStrategyInterface|SupportStrategyInterface $supportStrategy
```

Comment:
`SupportStrategyInterface` is deprecated. I think keeping only `WorkflowSupportStrategyInterface` here is fine.